src/test/java/uk/nhs/prm/e2etests/test/RepositoryE2ETest.java

```diff
@@ -555,4 +553,20 @@ private void setManagingOrganisationToRepo(String nhsNumber) {
             pdsAdaptorService.updateManagingOrganisation(nhsNumber, repoOdsCode, pdsResponse.getRecordETag());
         }
     }
+
+    @Test
+    void shouldSendUnexpectedMessageFormatsThroughToEhrTransferServiceDeadLetterQueue() {
+        String text = "Hello World!";
+        String sql = "SELECT * FROM Fragment";
+        String html = "<html><body><h1>This is html!</body></html>";
+        String binaryStream = "100110 111010 001011 101001";
```

Comment:
I think maybe we can remove this line or rename the variable name?  Because this actually is a string of numbers, and probably string and binary are treated as different object classes in java.